### ISSUE TITLE
`yara examine` crashes

### ISSUE THREAD

**Issue by Althorion** [Satellile/yara#1]
Running `yara examine` causes a program crash, with the following message/stack:
```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src\main.rs:316:52
stack backtrace:
   0:     0x7ff759ed8482 - <unknown>
   1:     0x7ff759efc7fb - <unknown>
   2:     0x7ff759ed323a - <unknown>
   3:     0x7ff759ed81cb - <unknown>
   4:     0x7ff759edad99 - <unknown>
   5:     0x7ff759edaa1b - <unknown>
   6:     0x7ff759edb4c8 - <unknown>
   7:     0x7ff759edb37b - <unknown>
   8:     0x7ff759ed9149 - <unknown>
   9:     0x7ff759edb070 - <unknown>
  10:     0x7ff759f80235 - <unknown>
  11:     0x7ff759f802ec - <unknown>
  12:     0x7ff759c2f510 - <unknown>
  13:     0x7ff759c2d389 - <unknown>
  14:     0x7ff759c28d5c - <unknown>
  15:     0x7ff759c3ab96 - <unknown>
  16:     0x7ff759c16b2c - <unknown>
  17:     0x7ff759ecc64e - <unknown>
  18:     0x7ff759c371bc - <unknown>
  19:     0x7ff759f636cc - <unknown>
  20:     0x7ff81e8a257d - BaseThreadInitThunk
  21:     0x7ff81f3eaa58 - RtlUserThreadStart
```

**Comment by Satellile** [comment]
Well, I'm super late on this one, rip.

Thanks for letting me know about the issue, it looks like it was caused by a ComfyUI update months ago. I'll take a look when I get a chance, and release a fix.

**Comment by Satellile** [comment]
It's fixed with the new release (v1.0.0).

It was caused by not having any KSampler nodes in a workflow (such as if you only used KSamplerAdvanced nodes). I didn't use those, so didn't think to account for them. [Commit](https://github.com/Satellile/yara/commit/78168a17c31dff1eb61f69b86094431991d5ac29)
